reana_server/reana_admin/check_workflows.py

```diff
@@ -30,38 +31,55 @@
 from reana_server.reana_admin.consumer import CollectingConsumer
 
 
-class WorkflowCheckFailed(Exception):
-    """Error when workflow doesn't pass some check."""
+class CheckFailed(Exception):
+    """Error when a check for workflow or session fails."""
 
 
-WorkflowCheckResult = Tuple[Workflow, List[WorkflowCheckFailed]]
+class FailedToCollect(Exception):
```

Comment:
```suggestion
class InfoCollectionError(Exception):
```

What do you think about this?